src/executable/extract_haplotype_kmers_count_parent_kmers.cpp

```diff
@@ -185,15 +185,15 @@ int main (int argc, char* argv[]){
             ->required();
 
     app.add_option(
-            "-p,--paternal_kmer_fa",
+            "-p,--paternal_kmer_FASTA",
             paternal_kmers,
-            "paternal kmers in .fa format")
+            "paternal kmers in FASTA format")
             ->required();
 
     app.add_option(
-            "-m,--maternal_kmer_fa",
+            "-m,--maternal_kmer_FASTA",
             maternal_kmers,
-            "maternal kmers in .fa format")
+            "maternal kmers in FASTA format")
```

Comment:
sorry I didn't mean to make any changes in the argument name, just in the description (help string)

Reply:
maybe just `--maternal_kmers`?

Reply:
ok I'll go with `maternal_kmers`